### DISCUSSION TITLE
Reordering

### DISCUSSION THREAD

**Discussion by roblevintennis** [community/community#9047] [Projects and Issues] [answered]
It would be lovely to be able to reorder the issues in both Issues tab, and when in Projects Beta (board or table views). The use case is I'm entering in all my tasks and I want to line them up in the order I plan to do them, but I'm sort of stuck with reverse ordering of how I entered them in.

**Comment by dotansimha** [comment]
Similar: https://github.com/github/feedback/discussions/7773

**Comment by bsutton** [comment, marked as answer]
Being able to re-order issues in the table view feels like a must have.
There is a hack around this (add a numeric field called ordinal/priority or similar) and sort by that column.
Still it would be nice to be able to drag and drop the oder.
